website/api_versioned_docs/version-1.0/blockchain-integration/contracts/verifier.md

```diff
@@ -47,16 +47,31 @@ You can find detailed information in the [version management design][VersionMana
 
 ### Contract Addresses
 
-You can find detailed information about all the contracts in this table in the [risc0-ethereum repo][risc0-ethereum-contracts]
-
-| Contract                                                           | Network            | Address                                                        |
-| ------------------------------------------------------------------ | ------------------ | -------------------------------------------------------------- |
-| [RiscZeroVerifierRouter][RiscZeroVerifierRouter.sol]               | [Ethereum Mainnet] | [0x8EaB2D97Dfce405A1692a21b3ff3A172d593D319][mainnet-router]   |
-| [RiscZeroVerifierEmergencyStop][RiscZeroVerifierEmergencyStop.sol] | [Ethereum Mainnet] | [0x44c220f0598345195cE99AD6A57aDfFcb9Ea33e7][mainnet-estop]    |
-| [RiscZeroGroth16Verifier][RiscZeroGroth16Verifier.sol]             | [Ethereum Mainnet] | [0xf70aBAb028Eb6F4100A24B203E113D94E87DE93C][mainnet-verifier] |
-| [RiscZeroVerifierRouter][RiscZeroVerifierRouter.sol]               | [Sepolia]          | [0x925d8331ddc0a1F0d96E68CF073DFE1d92b69187][sepolia-router]   |
-| [RiscZeroVerifierEmergencyStop][RiscZeroVerifierEmergencyStop.sol] | [Sepolia]          | [0xeA41bc40DAA1d0E681c7e8eA449b89a290e922e8][sepolia-estop]    |
-| [RiscZeroGroth16Verifier][RiscZeroGroth16Verifier.sol]             | [Sepolia]          | [0x3d24C84FC1A2B26f9229e58ddDf11A8dfba802d0][sepolia-verifier] |
+You can find detailed information about all the contracts in this table in the [risc0-ethereum repo][risc0-ethereum-contracts]. For your convenience, here is a quick recap:
+
+- RISC Zero provides official deployments on the chains listed below.
+- Each deployment includes a [router][RiscZeroVerifierRouter.sol]. This contract implements [IRiscZeroVerifier][IRiscZeroVerifier] and can be used to verify proofs from _any_ supported version of zkVM. New versions can only be added by RISC Zero via the [timelock][TimelockController.sol] governance mechanism. Addresses for the router and timelocks are given below.
+- In addition to the router and timelock, each of the supported chains also has a [fixed-version verifier][RiscZeroGroth16Verifier.sol] contract for each supported versions of zkVM. These fixed-version verifiers are wrapped by [emergency-stop mechanisms][RiscZeroVerifierEmergencyStop.sol] (which implement [IRiscZeroVerifier][IRiscZeroVerifier]); RISC Zero can use the emergency-stop to shutdown the verifier in the event that a soundness bug is ever discovered in that particular version. Addresses for the emergency-stop contracts can be fetched by querying the router (using one of the "selector" values listed below); addresses for the underlying fixed-version verifiers can be fetched by querying the respective emergency-stop.
+
+Fixed-version contracts:
+
+| zkVM version | Contract version ("selector") |
```

Comment:
This new table is confusing to me -- suggest adding words and/or links to clarify what this information means.

Reply:
Added some verbiage